tests/tests_pytorch/models/test_tpu.py

```diff
@@ -366,20 +325,32 @@ def test_if_test_works_with_checkpoint_false(tmpdir):
     assert trainer.state.finished, f"Training failed with {trainer.state}"
 
 
-@RunIf(tpu=True)
-def test_tpu_sync_dist():
-    """Test tpu spawn sync dist operation."""
+def wrap_launch_function(fn, strategy, *args, **kwargs):
+    # FIXME: i think this requires https://github.com/Lightning-AI/lightning/pull/11073
+    # the launcher does not manage this automatically. explanation available in:
+    # https://github.com/Lightning-AI/lightning/pull/14926#discussion_r982976718
+    # strategy.setup_environment()
+    return fn(*args, **kwargs)
+
 
-    def test_sync_dist(rank):
-        trainer = Trainer(accelerator="tpu", devices=8)
-        trainer.strategy._launched = True
+def xla_launch(fn):
+    strategy = TPUSpawnStrategy(parallel_devices=list(range(8)))
+    launcher = _XLALauncher(strategy=strategy)
+    wrapped = partial(wrap_launch_function, fn, strategy)
+    return launcher.launch(wrapped, strategy)
```

Comment:
You need to set the accelerator here. 
```suggestion
    strategy = TPUSpawnStrategy(parallel_devices=list(range(8)), accelerator=TPUAccelerator())
    launcher = _XLALauncher(strategy=strategy)
    wrapped = partial(wrap_launch_function, fn, strategy)
    return launcher.launch(wrapped, strategy)
```

Reply:
Done. I've accepted that we dont have lazy init yet.